concurrent_test.go

```diff
@@ -364,10 +366,23 @@ func executeRead(db *btesting.DB, bucket []byte, keys []string, readInterval dur
 	return rec, err
 }
 
-func executeWrite(db *btesting.DB, bucket []byte, keys []string, writeBytes bytesRange) (historyRecord, error) {
+func executeWrite(db *btesting.DB, bucket []byte, keys []string, writeBytes bytesRange, noopWriteRatio int) (historyRecord, error) {
 	var rec historyRecord
 
 	err := db.Update(func(tx *bolt.Tx) error {
+		if mrand.Intn(100) < noopWriteRatio {
+			// A no-op write transaction has two consequences:
+			//    1. The txid increases by 1;
+			//    2. Two meta pages point to the same root page.
+			rec = historyRecord{
+				OperationType: Write,
+				Key:           "",
```

Comment:
Please consider defining an explicit constant like: `const NoOpTxKey string = '%magic-no-op-key%'`.

This make understanding the code easier by searching where the constant is used. Searching for `` is not that unique.

Reply:
thx & Updated. PTAL